lib/Varien/Data/Form/Filter/Datetime.php

```diff
@@ -51,6 +55,10 @@ public function inputFilter($value)
      */
     public function outputFilter($value)
     {
+        if (!$value) {
```

Comment:
are this checks ok? we don't use this kind of notation since a while, shouldn't it be if $value === null?

Reply:
Value here is actually an empty string since it comes from form submission, so if you want to tighten the code a bit, I am happy to do the following check instead:

`if ($value === null || $value === '') `

Reply:
I'm wondering cause `if !$value` would also be positive in case of $value = 0

Reply:
Yes, agree `if ($value === null || $value === '')` would be a stronger check. I'll update it.

Reply:
iirc that code is from the mage2 pr. Didnt check if thats still present.

Reply:
> I'm wondering cause `if !$value` would also be positive in case of $value = 0

Just from reading ... 0 needs no further validation/checks/... Looks okay for me.

Reply:
Fully understand ... but 0 value would still cause unnecessary calls. At the end ... lets fix it.

Reply:
there are probably many more values that could be "managed" by us before going to the external library but at that point.. I think that, if there's a value, then it's the external library responsability.

Reply:
Yep. It will be fixed in zf1. If some calls can be avoided her ... why not? +1

Reply:
again, not to replicate the checks that are probably done by the external library. anyway.